go/src/meguca/parser/commands.go

```
@@ -63,6 +63,19 @@ func parseCommand(match []byte, board string) (com common.Command, err error) {
 			com.Pyu, err = db.GetPyu()
 		}
 
+	// Roulette
+	case bytes.Equal(match, []byte("roulette")):
+		com.Type = common.Roulette
+		max, err := db.DecrementRoulette()
+		if err != nil {
+			panic(err)
```

Nope, nope, nope. Please just return the error.